src/BloomTests/Book/BookTests.cs

```diff
@@ -940,7 +940,7 @@ public void Save_UpdatesMetadataIsbnAndPageCount()
 		}
 		
 		[Test,Ignore("Known bug: BL-2962")]
-		public void Save_UpdatesBookInfoMetadataTags()
+		public void Save_UpdatesBookInfoMetadataTags2()
```

Comment:
The new test came in as part of a fix for BL-2962, so continuing to ignore this test is inappropriate. I'm pretty sure one or the other should be deleted, since they are very similar. @hatton, which would you recommend? Assuming the old one now passes, I'd be inclined to delete the new one, to minimize diffs (and of course remove the ignore).